src/main/java/spoon/reflect/code/CtTypeAccess.java

```diff
@@ -25,4 +25,7 @@
  * 		type in the expression.
  */
 public interface CtTypeAccess<T> extends CtExpression<T> {
+	boolean isClassType();
```

Comment:
to be discussed. I'm not sure this have to explicit in the metamodel, it may be inferred from the context where it is used